### DISCUSSION TITLE
preset refresh issue

### DISCUSSION THREAD

**Discussion by joihn** [danny-avila/LibreChat#3548] [Troubleshooting]
### What happened?

if you modify a currently active preset, change are not taken into account until you deactivate/reactivate preset

### Steps to Reproduce

create a preset such as "say only banana"
activate it 
modify it "say only orange"
query it with a question

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Some of the wording for presets could be improved. The way it works is they get "applied" rather than remain "active." If you want the behavior of an "active" preset, you would have to "pin" one to be a default preset
